spec/features/platform_admin/platform_admin_can_visit_its_dashboard_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+describe 'when a platform admin visits its dashboard' do
+  before(:all) do
+    @platform_admin = create(:platform_admin, email: "platform_admin@example.com")
+    role = create(:role, name: "store_admin")
```

Comment:
You can remove lines 6 to 8- this is accounted for in the nested factory of :platform_admin, I pulled the branch down locally and tested it.